lib/streams/germanAbbreviationStream.js

```diff
@@ -0,0 +1,30 @@
+var _ = require('lodash');
+var through = require('through2');
+
+function expandGermanicStreetSuffixes(record) {
+  if (_.isEqual(record.parent.country_a, ['DEU']) || _.isEqual(record.parent.country_a,['CHE']) ||
+    _.isEqual(record.parent.country_a, ['AUT'])){
+    var temp = record.address_parts.street.replace(/([^\s]+)(s|S)tr\.?$/i,'$1$2trasse');
+    return temp;
+  }
+  if (_.isEqual(record.parent.country_a, ['NED'])){
+    return record.address_parts.street.replace(/([^\s]+)(s|S)tr\.?$/i,'$1$2traat');
+  }
+  if (_.isEqual(record.parent.country_a, ['DMK'])){
+    return record.address_parts.street.replace(/([^\s]+)(s|S)tr\.?$/i,'$1$2træde');
```

Comment:
Its probably best to discard the case of the `S` here and just return `foostrasse` instead of `fooStrasse`